src/python_tools/robot_broadcast_receiver.py

```diff
@@ -0,0 +1,31 @@
+from software.proto.announcement_pb2 import Announcement
```

Comment:
would be generally good to bring all constants out to the top and give them meaningful names